src/boot/efi/stub.c

```diff
@@ -34,8 +34,8 @@ enum {
         _INITRD_DYNAMIC_FIRST,
         INITRD_CREDENTIAL = _INITRD_DYNAMIC_FIRST,
         INITRD_GLOBAL_CREDENTIAL,
-        INITRD_SYSEXT,
         INITRD_CONFEXT,
+        INITRD_SYSEXT,
```

Comment:
hmm, this change does not look right. The order before ac32323271305252935469723e460139a65ed378 was sysext first, confext after. We should stick to that I think